packages/cdk/lib/stacks/core-stack.ts

```diff
@@ -80,6 +81,13 @@ export class CoreStack extends Stack {
       },
     });
 
+    new BucketDeployment(this, "WesAdapter", {
+      sources: [Source.asset(path.join(os.homedir(), ".agc", "wes"))],
+      destinationBucket: this.bucket,
+      destinationKeyPrefix: "wes",
+      prune: true,
```

Comment:
I tested it by running account activation a few times, (having the same thought that it would delete the files in the whole bucker), but it didn't. I am guessing this one deploys to a sub dir inside the bucker so it only cleans the .zip file which is our intention. But I will run a few more times today after I address all the other comments.

Reply:
Tested again, prune seems to only clean up the subdirectory as I have a destinationKeyPrefix here.